modules/seqexec/server/src/main/scala/seqexec/server/SeqTranslate.scala

```diff
@@ -394,7 +394,7 @@ class SeqTranslate(site: Site, systems: Systems, settings: Settings) {
   private def commonHeaders(config: Config, tcsSubsystems: List[TcsController.Subsystem], inst: InstrumentSystem)(ctx: ActionMetadata): Header =
     new StandardHeader(
       inst,
-      ObsKeywordReaderImpl(config, site.displayName.replace(' ', '-')),
+      ObsKeywordReaderImpl(config, site.longName.replace(' ', '-')),
```

Comment:
What's the meaning of changing `' '` to `'-'`? If we decide to change `longName` at some point will it break things? Do we need to add a `tcsValue` or something?

Reply:
You're right we should have the actual values somewhere else. Given it is only used here, it could live in the seqexec rather than being a part of `Site`

Reply:
I could argue it either way I guess. Lots of things in the model will need TCC values and it wouldn't bother me to just define them in the model and be done with it. But as you say most of the time we don't care and we could define them somewhere else.